### DISCUSSION TITLE
Introduce a technical level to feed items

### DISCUSSION THREAD

**Discussion by Metagrids** [dailydotdev/daily#454] [Feature Requests]
Please introduce a technical level to feed items. Now a mixture of all levels from novice to very expert is shown to me. Lets users filter what they want based on keyword and technical level.

**Comment by rebelchris** [comment]
Hi @Metagrids this is a really valid idea, and we have this on our roadmap.
Thanks for noting it down, we'll keep you posted 👀

**Reply by Metagrids** [reply]
Thanks Chris!
